Simplenote/Classes/SignupRemote.swift

```diff
@@ -4,12 +4,9 @@ import Foundation
 //
 class SignupRemote: Remote {
     func signup(with email: String, completion: @escaping (_ result: Result<Data?, RemoteError>) -> Void) {
-        guard let request = request(with: email) else {
-            completion(.failure(RemoteError.urlRequestError))
-            return
-        }
+        let urlRequest = request(with: email)!
```

Comment:
Let's force-unwrap URL and return non-optional request here. If we're gonna crash, we should crash as close to the initial source as possible.